src/pages/ServiceGraph/ServiceGraphRouteHandler.tsx

```diff
@@ -88,8 +76,26 @@ export default class ServiceGraphRouteHandler extends React.Component<
         edgeLabelMode: nextEdgeLabelMode
       };
       sessionStorage.setItem(SESSION_KEY, JSON.stringify(newParams));
-      this.setState({ ...newParams });
+      return { ...newParams };
     }
+    return {};
```

Comment:
I think you should return null here:

> It should return an object to update the state, or null to update nothing.

see https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops

Reply:
You are right , Changed !!! Thanks @josejulio